script/build.sh

```diff
@@ -27,35 +30,46 @@ BUILDS="$WORKINGDIR/interpolation/builds";
 [ -d $BUILDS ] || mkdir -p $BUILDS;
 
 # a directory where this specific build will live
-export BUILDDIR="$BUILDS/$TIMESTAMP";
+export BUILDDIR="${BUILDDIR:-"$BUILDS/$TIMESTAMP"}";
 [ -d $BUILDDIR ] || mkdir -p $BUILDDIR;
 
 # location of temp files
 export SQLITE_TMPDIR="$BUILDDIR/tmp"; # a directory with enough free space to store sqlite tmp files
 export PBF2JSON_TMPDIR="$BUILDDIR/tmp/leveldb"; # a directory with enough free space to store leveldb tmp files
 
+echo "$BUILDDIR"
+echo "$WORKINGDIR"
+
 # run polyline importer
+echo "- importing polylines"
 $DIR/import.sh;
 
 # archive street database (using parallel gzip when available)
+echo "- archiving street database"
 if type pigz >/dev/null
   then pigz -k -c --best "$BUILDDIR/street.db" > "$BUILDDIR/street.db.gz";
   else gzip -c --best "$BUILDDIR/street.db" > "$BUILDDIR/street.db.gz";
 fi
 
 # run openaddresses conflation
+echo "- conflating openaddresses"
 $DIR/conflate_oa.sh;
 
 # run openstreetmap conflation
+echo "- conflating openstreetmap"
 $DIR/conflate_osm.sh;
 
 # run tiger conflation
+echo "- conflating tiger"
 $DIR/conflate_tiger.sh;
 
 # run vertex interpolation
+echo "- interpolating vertices"
 $DIR/vertices.sh;
 
 # archive address database (using parallel gzip when available)
+echo "- archiving address database"
```

Comment:
Adding all these is great. I often was confused when running the build what state it was in